SMPyBandits/Policies/RAWUCB.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import division, print_function  # Python 2 compatibility
```

Comment:
Missing docstring for the module! Please add some documentation.

Reply:
The paper is not aired yet.

Reply:
Okay, but you can add a comment (a docstring for the whole file) saying it's a work in progress in 2019 by Author1, Author2 etc.